client/evm-tracing/src/listeners/call_list.rs

```diff
@@ -79,6 +79,12 @@ pub struct Listener {
 	/// True if only the `GasometerEvent::RecordTransaction` event has been received.
 	/// Allow to correctly handle transactions that cannot pay for the tx data in Legacy mode.
 	record_transaction_event_only: bool,
+
+	/// If true the listener will collect EvmEvent::Log events.
+	pub with_log: bool,
+
+	/// Storage for all logs emitted by EvmEvent::Log events.
+	pub log_entries: BTreeMap<u32, Vec<Log>>,
```

Comment:
Why to you use another map? Can you put the logs in the `entries` map instead? (By adding the log variant in Call enum)

Reply:
Good point! I will update the code now

Reply:
Pushed a partially implemented version of log tracing  [here](https://github.com/moonbeam-foundation/moonbeam/tree/aleks-debug-trace-call-options-logs-in-entries). However, the log addition-related unit tests seem to fail as a result of the changes